cql3/restrictions/restriction.hh

```diff
@@ -93,7 +93,7 @@ struct token {};
 
 /// Operator restriction: LHS op RHS.
 struct binary_operator {
-    std::variant<std::vector<column_value>, token> lhs;
+    std::variant<column_value, std::vector<column_value>, token> lhs;
```

Comment:
It will be good to further disambiguate std::vector<column_value> as a new struct column_value_tuple, later on. This makes the role of the vector visible in the name.